Src/bat_charge.c

```diff
@@ -0,0 +1,96 @@
+/*
+ * bat_charge.c
+ *
+ *  Created on: Sep, 26 2020
+ *      Author: Luke Oxley
+ */
+#include "bat_charge.h"
+
+uint8_t temp_array[READ_MSG_SIZE];
+uint8_t read_byte = 0;
+uint8_t write_data[WRITE_MSG_SIZE + 1];
+
+void init_BQ21062(uint8_t * write_data);
+
+void task_manage_charger() 
+{
+    uint8_t currentF, voltageF, tempF;
+
+    while(1)
+    {
+        read_byte = set_address(STAT0_ADDR, READ_ENABLE);
+        HAL_I2C_Master_Receive(&hi2c1, read_byte, &temp_array[0], READ_MSG_SIZE, 0xFFFF);
+        while (hi2c1.State != HAL_I2C_STATE_READY)
+	    {
```

Comment:
curly braces need to be aligned with the first character of the statement they are encapsulating for